modules/openxr/openxr_api.h

```diff
@@ -46,17 +46,18 @@
 #include "core/templates/rb_map.h"
 #include "core/templates/rid_owner.h"
 #include "core/templates/vector.h"
+#include "servers/rendering_server.h"
 #include "servers/xr/xr_pose.h"
 
 #include <openxr/openxr.h>
 
-// Note, OpenXR code that we wrote for our plugin makes use of C++20 notation for initializing structs which ensures zeroing out unspecified members.
-// Godot is currently restricted to C++17 which doesn't allow this notation. Make sure critical fields are set.
-
 // forward declarations, we don't want to include these fully
 class OpenXRInterface;
 
 class OpenXRAPI {
+protected:
+	// _THREAD_SAFE_CLASS_
```

Comment:
If we don't want this (it's commented out), let's delete it completely

Reply:
Owh yeah, was just testing this, needs to be removed :)